ext/enchant/enchant.stub.php

```diff
@@ -1,11 +1,16 @@
 <?php
 
-/** @generate-function-entries */
+/**
+ * @generate-function-entries
+ * @generate-class-entries
```

Comment:
Hm, actually, shouldn't we make generate-class-entries imply generate-function-entries? I don't think class entry generation can work without also having function entry generation...

Reply:
Yes, there is already a check for this case (when `@generate-function-entries` is missing but `@generate-class-entries` is enabled) in gen_stub. But yes, It's reasonable to do what you've just suggested.

Reply:
Just to make sure before I commit it: would you prefer it if enabling `@generate-class-entries` would implicitly enable `@generate-function-entries` as well, right?

Reply:
Right.